models/weight_recorder.h

```diff
@@ -71,6 +73,8 @@ synapses that fulfill the given criteria.
 
    >>> nest.Connect(pre, post, syn_spec="stdp_synapse_rec")
 
+.. include:: ../models/recording_device.rst
+
 EndUserDocs */
```

Comment:
Add a See-also section to actually refer to other recorders
```suggestion
.. include:: ../models/recording_device.rst

See also
++++++++

EndUserDocs */
```